lib/infrastructure/logger.js

```diff
@@ -2,4 +2,7 @@ module.exports = {
   info({ event, app, data }) {
     console.log(JSON.stringify({ event, app, data }));
   },
+  error({ task, error, message }) {
+    console.log(JSON.stringify({ message, error, task }));
```

Comment:
```suggestion
    console.log(JSON.stringify({ status: 'ERROR', message, error, task }));
```
Pour le parsing automatique en erreur, on peut utiliser un attribut réservé par Datadog : https://docs.datadoghq.com/fr/logs/log_configuration/attributes_naming_convention/#attributs-réservés